### DISCUSSION TITLE
X button on Top-Right does nothing LOL

### DISCUSSION THREAD

**Discussion by leoheck** [community/community#38730] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

X button on Top-Right does nothing LOL.
Useless shit. 

![image](https://user-images.githubusercontent.com/1277920/200959211-955a9b85-e9bd-4bc2-a3a5-e0bb20a9c3cb.png)

**Comment by mikekavouras** [comment]
Thanks for the kind feedback, @leoheck. This shouldn't be an issue anymore since that announcement no longer appears on the home page.

**Reply by leoheck** [reply]
Np! Purging the useless content is even better. Thanks.
